Allow disabling HTTP request logs in dev server (#74349)
vercel/next.js: docs/01-app/04-api-reference/05-config/01-next-config-js/logging.mdx

}
```

### Incoming Requests

> By default all the incoming requests will be logged in the console during development. You can use this `incomingRequests` option and decided which requests to ignore.

Reviewer: ```suggestion
By default all the incoming requests will be logged in the console during development. You can use the `incomingRequests` option and decided which requests to ignore.
```

> Since it's only logging for development so this option doesn't affect the production build.

Reviewer: ```suggestion
Since this is only logged in development, this option doesn't affect production builds.
```

```js filename="next.config.js"
module.exports = {
  logging: {
    incomingRequests: {
      ignore: [/\api\/v1\/health/],
    },
  },
}
```

> Or you can disable only incoming requests logging by setting `incomingRequests` to `false`.

Reviewer: ```suggestion
Or you can disable incoming request logging by setting `incomingRequests` to `false`.
```
